5.6/alpine/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM alpine:3.3
+
+ENV PHP_INI_DIR /usr/local/etc/php
+RUN mkdir -p $PHP_INI_DIR/conf.d
+
+ENV PHP_VERSION 5.6.18
+ENV PHP_FILENAME php-5.6.18.tar.xz
+ENV PHP_SHA256 54dd9106c3469bc7028644d72ac140af00655420bbaaf4a742a64e9ed02ec1b0
```

Comment:
I'd suggest to use the form:

````
ENV PHP_VERSION=5.6.18 \
         PHP_FILENAME=php-5.6.18.tar.xz \
         PHP_SHA256=54dd9106c3469bc7028644d72ac140af00655420bbaaf4a742a64e9ed02ec1b0
````

Just so as to reduce the number of layers (one layer means an additional download after all).